znet/ws_connection.go

```diff
@@ -536,8 +514,13 @@ func (c *WsConnection) callOnConnStop() {
 }
 
 func (c *WsConnection) IsAlive() bool {
-	if c.isClosed {
+	if c.ctx == nil {
+		return false
+	}
+	select {
+	case <-c.ctx.Done():
```

Comment:
```suggestion
	return c.ctx.Err() != nil 
```
是否这样更好点, 可以更改 这个对象的 `IsClose` 函数，相关判断都走`IsClose`